Source/Csla/Configuration/Fluent/DataPortalClientOptions.cs

```diff
@@ -0,0 +1,59 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="DataPortalClientOptions.cs" company="Marimer LLC">
+//     Copyright (c) Marimer LLC. All rights reserved.
+//     Website: https://cslanet.com
+// </copyright>
+// <summary>Client-side data portal options.</summary>
+//-----------------------------------------------------------------------
+
+namespace Csla.Configuration
+{
+  /// <summary>
+  /// Client-side data portal options.
+  /// </summary>
+  public class DataPortalClientOptions
+  {
+    /// <summary>
+    /// Gets or sets a value indicating whether objects should be
+    /// automatically cloned by the data portal Update()
+    /// method when using a local data portal configuration.
+    /// </summary>
+    public bool AutoCloneOnUpdate
+    {
+      get
+      {
+        bool result = true;
+        string setting = ConfigurationManager.AppSettings["CslaAutoCloneOnUpdate"];
```

Comment:
I'm imagining the continued use of AppSettings here is an intentional decision - that there is a reason why this cannot, or should not, be defined in the startup code at compile time.

Reply:
Removed appsettings throughout